aspnet/getting-started/choosing-the-right-dotnet.rst

```diff
@@ -1,37 +1,65 @@
 
-The CoreCLR runtime (Microsoft.CoreCLR) and CoreFX libraries are distributed via `NuGet <https://www.nuget.org>`_. Because .NET Core has been built as a componentized set of libraries you can limit the API surface area your application uses to just the pieces you need. You can also run .NET Core based applications on much more constrained environments (ex. `Windows Server Nano <http://blogs.technet.com/b/windowsserver/archive/2015/04/08/microsoft-announces-nano-server-for-modern-apps-and-cloud.aspx>`_).
+.NET Core consists of:
 
-The API factoring in .NET Core was updated to enable better componentization. This means that existing libraries built for the .NET Framework generally need to be recompiled to run on .NET Core. The .NET Core ecosystem is relatively new, but it is rapidly growing with the support of popular .NET packages like JSON.NET, AutoFac, xUnit.net and many others.
+- **CoreFX**: a set of libraries
+- **CoreCLR**: (``Microsoft.CoreCLR``) a small, modular optimized runtime
 
-Developing on .NET Core allows you to target a single consistent platform that can run on multiple platforms. However, the .NET Core support for Mac and Linux is still very new and not ready for production workloads.
+The CoreCLR runtime and CoreFX libraries are distributed via `NuGet <https://www.nuget.org>`_. 
+
+The API factoring in .NET Core was architected for componentization. This means that existing libraries built for the .NET Framework generally need to be recompiled to run on .NET Core. The ecosystem is relatively new, but it is rapidly growing with the support of popular .NET packages like `JSON.NET <https://github.com/JamesNK/Newtonsoft.Json>`__, `AutoFac <http://autofac.org/>`__, `xUnit <https://github.com/xunit/xunit>`__  and many others.
+
+Developing on .NET Core allows you to target a single consistent platform that can run on multiple platforms. 
 
 Please see :doc:`/conceptual-overview/dotnetcore` for more details on what .NET Core has to offer.
 
 Mono
 ----
 
-`Mono <http://mono-project.com>`_ is a port of the .NET Framework built primarily for non-Windows platforms. Mono is open source and cross-platform. It also shares a similar API factoring to the .NET Framework, so many existing managed libraries work on Mono today. Mono is not a supported platform by Microsoft. It is however a good proving ground for cross-platform development while cross-platform support in .NET Core matures.
+`Mono <http://mono-project.com>`_ is a port of the .NET Framework built primarily for non-Windows platforms. Mono is open source and cross-platform. It also shares a similar API factoring to the .NET Framework, so many existing managed libraries work on Mono today. Mono is not a supported platform by Microsoft. It's a good proving ground for cross-platform development while .NET Core matures.
+
+Related Resources
+-----------------
 
-Summary
--------
+- :doc:`/fundamentals/index`
```

Comment:
Neither of these are really related to this topic. You could add a link to the DNX docs.